package/osx/fix-library-paths-electron.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+#
+# fix-library-paths.sh
```

Comment:
Comment doesn't match the actual filename (`fix-library-paths.sh` vs `fix-library-paths-electron.sh`).

Reply:
Resolved